src/forms/StudyFormSteps.js

```diff
@@ -48,6 +49,7 @@ export const InfoStep = ({
         content="Please provide the study's full name and a shortened version that may be used for display purposes."
       />
       <FormField
```

Comment:
alot of the fields in this form look very similar let's see if we can map them instead of repeated them to make each step file a little smaller